services/director-v2/tests/integration/02/test_dynamic_sidecar_nodeports_integration.py

```diff
@@ -258,7 +266,9 @@ async def db_manager(aiopg_engine: aiopg.sa.engine.Engine) -> DBManager:
     return DBManager(aiopg_engine)
 
 
-@pytest.fixture(scope="session", params=["true", "false"])
+# FIXME: ANE you can change this to something that runs everywhere. Thanks.
+# @pytest.fixture(scope="session", params=["true", "false"])
+@pytest.fixture(scope="session", params=["false"])
```

Comment:
I have a fix for this this. When se to `True` the test will be skipped if rclone plugin is not installe don the system.

Reply:
yes that is great